velero-examples/main.go

```diff
@@ -17,31 +17,31 @@ limitations under the License.
 package main
 
 import (
-	veleroplugin "github.com/heptio/velero/pkg/plugin"
+	veleroplugin "github.com/heptio/velero/pkg/plugin/framework"
 	"github.com/sirupsen/logrus"
 )
 
 func main() {
-	veleroplugin.NewServer(veleroplugin.NewLogger()).
-		RegisterBackupItemAction("backup-plugin", newBackupPlugin).
-		RegisterObjectStore("file", newFileObjectStore).
-		RegisterRestoreItemAction("restore-plugin", newMyRestorePlugin).
-		RegisterBlockStore("example-blockstore", newNoOpBlockStore).
+	veleroplugin.NewServer().
+		RegisterBackupItemAction("example/backup-plugin", newBackupPlugin).
```

Comment:
I think I'd make these `example.io/...` (or whatever your favorite TLD is)

Reply:
Waiting to finalize the format we're gonna go with.

Reply:
I think based on the discussion in the other PR, updating this to `example.io` or something similar still makes sense